### ISSUE TITLE
Upload large file The uploaded fragment is not contiguous with the last one

### ISSUE THREAD

**Issue by apurbajnu** [OneDrive/onedrive-api-docs#1273]
1st session uploaded. 2nd session never though I am sending the expected range. Please let me know what's this issue here? 

array(2) {
  ["postBody"]=>
  object(GuzzleHttp\Psr7\Stream)#6 (7) {
    ["stream":"GuzzleHttp\Psr7\Stream":private]=>
    resource(8) of type (stream)
    ["size":"GuzzleHttp\Psr7\Stream":private]=>
    NULL
    ["seekable":"GuzzleHttp\Psr7\Stream":private]=>
    bool(true)
    ["readable":"GuzzleHttp\Psr7\Stream":private]=>
    bool(true)
    ["writable":"GuzzleHttp\Psr7\Stream":private]=>
    bool(true)
    ["uri":"GuzzleHttp\Psr7\Stream":private]=>
    string(10) "php://temp"
    ["customMetadata":"GuzzleHttp\Psr7\Stream":private]=>
    array(0) {
    }
  }
  ["headers"]=>
  array(2) {
    ["Content-Length"]=>
    int(5242880)
    ["Content-Range"]=>
    string(23) "bytes 0-5242879/8255206"
  }
}
array(2) {
  ["expirationDateTime"]=>
  string(24) "2020-05-10T14:46:37.498Z"
  ["nextExpectedRanges"]=>
  array(1) {
    [0]=>
    string(15) "5242880-8255205"
  }
}
array(2) {
  ["postBody"]=>
  object(GuzzleHttp\Psr7\Stream)#6 (7) {
    ["stream":"GuzzleHttp\Psr7\Stream":private]=>
    resource(22) of type (stream)
    ["size":"GuzzleHttp\Psr7\Stream":private]=>
    NULL
    ["seekable":"GuzzleHttp\Psr7\Stream":private]=>
    bool(true)
    ["readable":"GuzzleHttp\Psr7\Stream":private]=>
    bool(true)
    ["writable":"GuzzleHttp\Psr7\Stream":private]=>
    bool(true)
    ["uri":"GuzzleHttp\Psr7\Stream":private]=>
    string(10) "php://temp"
    ["customMetadata":"GuzzleHttp\Psr7\Stream":private]=>
    array(0) {
    }
  }
  ["headers"]=>
  array(2) {
    ["Content-Length"]=>
    int(3012326)
    ["Content-Range"]=>
    string(29) "bytes 5242880-8255205/8255206"
  }
}
string(1205) "Client error: `PUT https://api.onedrive.com/rup/8653f7cfbe8f5ceb/eyJSZXNvdXJjZUlEIjoiODY1M0Y3Q0ZCRThGNUNFQiE4NjY5OSIsIlJlbGF0aW9uc2hpcE5hbWUiOiJzY3JyLm1vdiJ9/4my5v0-oqYXPv_hIdc3LcYaZJmfyvixEPQnqsKNodbUEAg5I91U1oB6afvzKdyUR6tmCQqDheoU59dkBmUdAlU4vesGELKuDp0BEmcEJIEOnQ/eyJuYW1lIjoic2Nyci5tb3YiLCJkZXNjcmlwdGlvbiI6IiIsIkBuYW1lLmNvbmZsaWN0QmVoYXZpb3IiOiJyZW5hbWUifQ/4wFN_ylnu7E1H-7J8S9Sh4gYzKzPUSIPD_CFEzV3NwkfR77b34G4TxFmNADPppVdbwKcmdvGRuGTEMAaeHUH6esnnFBBCHnDVJkP7Afl00dAdbF0ffTpbaPCXc6OZo7Hc6Ym43ZoD3e1bGbfEOmFSpKX7zz--Sv7V-v_57h4qSxX3AkHCSfFGeMv0faDEIQBeOMnZkf-_ADQUymncGvALpUXTWz4n7tNX4TzVLCUGgyGYI-jLQfoxk_1DUNhwByC2sNgkfD9ss2YERJBmfzw2BvIs2sA-B3ACyBT1r0CoShk_FR-Z7ko6wMbfYEcpJLnXkAEZQxEj_aMMuNggCOnGhIzdQny76alEcorHkLXICYfeowfFYD1j40Pmrfb8oKEQoJb8d8tUBv835BGBRkoS9MbywWDmrG0yw-T3rdXKQ0FjDTI0qAfj4-YXdbaXNipjV0dS6Y9FpUfXChiz11anqIlAIlPHaxQN9Y0RITv8acnG0OVJzzaCNFBL4YtBA48Wv_SoXQaTv_A87i8DbVE7GuMOMYOI68rF6aoeZ7QFtIlJrSP4r4zaux5OINkTaVyvYQLNlH7Bqb6ofpaAGuT5X2Sq4SvEEz0LNv8trsT6v75iLTzX5SSvMoi13J3XYRCHk` resulted in a `416 Requested Range Not Satisfiable` response:
{"error":{"code":"invalidRange","message":"The uploaded fragment is not contiguous with the last one.","innererror":{"co (truncated...)
"
int(416)

**Comment by k-tsoi** [comment]
@apurbajnu are you sending these in parallel or in sequence?

**Comment by patrick-rodgers** [comment]
As part of a repository clean up effort we are closing older issues. If this issue remains, please: open a new issue, reference this issue, and provide any additional details that may help in resolution. Thank you for your understanding as we work to improve our responsiveness.
